release.json

```diff
@@ -44,34 +44,34 @@
         "WINDOWS_APMINJECT_SHASUM": "5fdd62a84e640204386b9c28dc2e3ac5d9b8adde6427cb9f5914619f94d7b5bd"
     },
     "release-a6": {
-        "INTEGRATIONS_CORE_VERSION": "7.56.0-rc.2",
-        "OMNIBUS_SOFTWARE_VERSION": "7.56.0-rc.3",
-        "OMNIBUS_RUBY_VERSION": "7.56.0-rc.1",
+        "INTEGRATIONS_CORE_VERSION": "7.61.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.61.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.61.0-rc.1",
         "JMXFETCH_VERSION": "0.49.6",
         "JMXFETCH_HASH": "f06bdac1f8ec41daf9b9839ac883f1865a068b04810ea82197b8a6afb9369cb9",
         "SECURITY_AGENT_POLICIES_VERSION": "v0.61.0",
-        "MACOS_BUILD_VERSION": "6.56.0-rc.3",
+        "MACOS_BUILD_VERSION": "7.61.0-rc.1",
```

Comment:
Unrelated to my review scope, but is this change from 6.x to 7.x in `release-a6` expected?

Reply:
It is not but it's not used anymore. I'll revert it and fill a card to fix the tool. Thank you